checkov/common/output/csv.py

```diff
@@ -86,10 +91,15 @@ def add_sca_package_resources(self, resource: Record | ExtraResource, git_org: s
                 "Git Repository": git_repository,
                 "Vulnerability": resource.vulnerability_details.get("id"),
                 "Severity": severity,
+                "Description": resource.vulnerability_details.get("description"),
                 "Licenses": resource.vulnerability_details.get("licenses"),
             }
```

Comment:
how about moving this dict to a separate variable, ex. `csv_table_entry = {}` then you can check for `CHECKOV_DISPLAY_REGISTRY_URL` and add the `registry_url` to the entry and after that add the entry to the table.

Reply:
not sure I got it
can you give an example?